shell/platform/embedder/embedder_surface_metal.mm

```diff
@@ -44,7 +44,8 @@
   }
 
   const bool render_to_surface = !external_view_embedder_;
-  auto surface = std::make_unique<GPUSurfaceMetalSkia>(this, main_context_, render_to_surface);
+  auto surface = std::make_unique<GPUSurfaceMetalSkia>(this, main_context_, MsaaSampleCount::kNone,
+                                                       render_to_surface);
```

Comment:
Specifically, in the previous version of this patch I wrote

```cpp
auto surface = std::make_unique<GPUSurfaceMetalSkia>(this, main_context_, render_to_surface, 1);
```

Which meant `render_to_surface` was _always_ true, which is the wrong thing to do (and caused the blank screen rendering).

Reply:
Using the enum everywhere means that wouldn't even compile now.